Assets/MRTK/Tools/MSBuild/Scripts/UnityProjectInfo.cs

```diff
@@ -25,7 +25,13 @@ public class UnityProjectInfo
         {
             "Windows.UI.Input.Spatial",
             "LeapMotion",
-            "LeapMotion.LeapCSharp"
+            "LeapMotion.LeapCSharp",
+#if UNITY_2019_3_OR_NEWER
+            "Oculus.VR",
+            "Oculus.VR.Editor",
+            "Unity.XR.Oculus",
+            "Unity.XR.WindowsMixedReality"
+#endif
```

Comment:
Should we try to ensure these packages are resolved, and build the corresponding MRTK binaries with the right `#defines`? Otherwise, we might miss some classes/methods, like https://github.com/microsoft/MixedRealityToolkit-Unity/blob/067a14ef05f01bcb540be396f4f8a91579ff0aea/Assets/MRTK/Providers/Oculus/XRSDK/MRTK-Quest/Scripts/Input/Controllers/OculusHand.cs#L102-L107

Reply:
I am open to revisit the decision, but that was based on the following:
1. I noticed even without those external packages the docs generated still contain classes like OculusHand (although arguably without some functions defined within #define.
2. The docs generated without those packages matches what we have on github.io right now
3. As we continue to grow MRTK there may be more cases where #defines pops up and I think trying to include more and more packages for docs generation might not be feasible, and there is also the possibility where two packages we need in order to generate the full doc do not play well with each other. Thus, my proposal would be trying to expose the public functions/properties outside the #defines when possible and add a remarks section to classes where the first option is not available saying please refer to the source for platform dependent functions.

Reply:
Seems reasonable! I think there will be some efforts to update our pipelines to properly test the Leap and Oculus import and build paths, so we might be able to leverage any work there here too